app/logic/redis_cached_call.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+class RedisCachedCall
+  def self.cache(key, bust_cache, &block)
+    if bust_cache
+      RedisService.set(key, nil)
```

Comment:
Ik vind dit geen goede oplossing. Als bust_cache aan staat, en je wilt wel een waarde opslaan, dan roep je RedisService.set twee keer direct achter elkaar aan, één keer om `key` op nil te zetten en direct erna om `key` op de waarde te zetten waar je m eigenlijk op wilt zetten. Die eerste call is dan overbodig lijkt me?